src/vs/workbench/contrib/interactiveEditor/browser/interactiveEditorActions.ts

```diff
@@ -547,3 +548,21 @@ export class ContractMessageAction extends AbstractInteractiveEditorAction {
 		ctrl.updateExpansionState(false);
 	}
 }
+
+export class AccessibilityHelpEditorAction extends EditorAction {
```

Comment:
Please use `EditorAction2` has other actions do in this file

Reply:
Yeah I tried and it didn't work - there was an error being thrown related to the number of lines exceeding max line count, will investigate